### ISSUE TITLE
accessibility text not updating

### ISSUE THREAD

**Issue by matt121113** [pmotschmann/Evolve#232]
I'm not sure if this happens anywhere else, but when assigning workers, NVDA does not read new values until you go out then back into the screen.
An example is. I have 5 unemployed workers. I assign three to farming, and two to lumber. If I don't exit and reenter that screen, I am still told by NVDA that I've got 5 free workers.

**Comment by obert01** [comment]
For your information, I am encountering this issue with Google Chrome, but it works perfectly with Firefox.

**Comment by sukiletxe** [comment]
If I use object navigation in that screen it works correctly.

**Comment by pmotschmann** [comment]
Haven't heard this complaint in a while, I think it got resolved
